### DISCUSSION TITLE
GH has a new layout!

### DISCUSSION THREAD

**Discussion by donaldli2020** [ProdigyPNP/ProdigyMathGameHacking#20] [Forum]
>

**Comment by afkvido** [comment]
yes

**Reply by donaldli2020** [reply]
there are rounded rectangles on hover on each repo tab
code issues 3 pull requests 1 discussions actions projects wiki security insights

**Reply by donaldli2020** [reply]
if ur not gonna use wiki turn it off

**Reply by afkvido** [reply]
the **Wiki** is there to redirect people to the wiki in the repository.
